core/controllers/suggestion.py

```diff
@@ -1186,8 +1183,14 @@ def _construct_exploration_suggestions(
     exp_id_to_exp = exp_fetchers.get_multiple_explorations_by_id(list(exp_ids))
     for suggestion in suggestions:
         exploration = exp_id_to_exp[suggestion.target_id]
-        content_html = exploration.get_content_html(
-            suggestion.change.state_name, suggestion.change.content_id)
+        content_html = ''
+        try:
+            content_html = exploration.get_content_html(
+                suggestion.change.state_name, suggestion.change.content_id)
+        except ValueError:
+            # If the exploration content is no longer available, return the
+            # suggestion's content.
+            content_html = suggestion.change.content_html
```

Comment:
Conceptually, isn't this weird -- shouldn't it be the empty string or None to denote that the content in the original exploration no longer exists?